ci-operator/step-registry/ipi/conf/gcp/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
+- stevekuznetsov
```

Comment:
Please no

Reply:
I am blindly copying from the parent directory, because some CI presubmit required per-directory `OWNERS` here.   If you have suggested subsets or different sets or whatever.  Or just want me to set an empty/dummy `OWNERS` to make the presubmit job happy, let me know.

Reply:
>  If you have suggested subsets or different sets or whatever.

What I really want here is access to [the installer's `gcp-approvers` alias][1], but since we're no longer mirroring aliases into this repo ([here][2] and, internally, [here][3]), I can't do that.

[1]: https://github.com/openshift/installer/blob/9bb6e40a6840d9d98c67e6bc40c1dd49d69dd2de/OWNERS_ALIASES#L51
[2]: https://github.com/openshift/ci-tools/pull/115#discussion_r350000636
[3]: https://issues.redhat.com/browse/DPTP-690